app/views/users/show.html.erb

```diff
@@ -1 +1,53 @@
-<div>I should show user info for one user. Probably in some dl, (dt/dd)* type list. Please fix me.</div>
+<h1> <%= @user.name %> </h1>
+<%= @user.name %><br>
+<%= @user.email %><br>
+<%= @user.phone_number %><br>
+<%= @user.created_at %><br>
+<%= @user.role.humanize %><br>
+<% if @user.deactivated? %>
+  Inactive
+
+  <%= link_to "Activate", deactivate_index_path(id: @user.id), method: :post,
```

Comment:
This deactivation stuff needs an overhaul but that is probably outside the scope of this PR.